static/js/map/map.gym.js

```diff
@@ -382,33 +382,33 @@ function gymLabel(gym) {
                   <div id='raid-container-right'>
                     <div class='title ongoing'>
                       <div>
-                        ${name} <i class="fas ${genderClasses[raid.gender - 1]}"></i> #${raid.pokemon_id} Raid
+                        ${i8ln('Raid')}: ${name} <i class="fas ${genderClasses[raid.gender - 1]}"></i> #${raid.pokemon_id}
                       </div>
                     </div>
                     <div class='disappear'>
                       ${timestampToTime(raid.end)} (<span class='label-countdown' disappears-at='${raid.end}'>00m00s</span>)
                     </div>
                     <div class='info-container'>
                       <div>
-                        CP: <strong>${raid.cp}</strong>
+                        ${i8ln('CP')}: <strong>${raid.cp}</strong>
                       </div>
                       <div>
-                        Fast: <strong>${fastMoveName}</strong> <img class='move-type-icon' src='static/images/types/${fastMoveType.toLowerCase()}.png' title='${i8ln(fastMoveType)}' width='15'>
+                        ${i8ln('Fast move')}: <strong>${fastMoveName}</strong> <img class='move-type-icon' src='static/images/types/${fastMoveType.toLowerCase()}.png' title='${i8ln(fastMoveType)}' width='15'>
                       </div>
                       <div>
-                        Charge: <strong>${chargeMoveName}</strong> <img class='move-type-icon' src='static/images/types/${chargeMoveType.toLowerCase()}.png' title='${i8ln(chargeMoveType)}' width='15'>
+                        ${i8ln('Charge move')}: <strong>${chargeMoveName}</strong> <img class='move-type-icon' src='static/images/types/${chargeMoveType.toLowerCase()}.png' title='${i8ln(chargeMoveType)}' width='15'>
```

Comment:
Charge was ambiguous, it can either be the move `Charge` or the label for a charge move (here): they have different translations. I understand the need to keep the popup tidy, so I can revert that and live with the change :)

Would you accept a better fix later with a second parameter to `i8ln` to lookup a custom string in the locale file? eg `i8ln('Charge', 'move_charge')` and `i8ln('Charge', 'label_charge'}`

Reply:
Lets revert for now and as you said fix it later. I was thinking before to use keys for all translation to prevent issues like these.